tests/helpers/providers/record_provider.py

```diff
@@ -48,3 +48,16 @@ def record(self, data=None, with_control_number=False):
             record.update(data)
         schema_validate(record)
         return record
+
+    def author(self, data=None, with_control_number=False):
```

Comment:
I'll integrate this part on the ``record`` it's duplicate code which only changes ``_collections`` and ``$schema``